data/operators/power/line.json

```
@@ -4878,21 +4878,22 @@
       }
     },
     {
-      "displayName": "PJM",
-      "id": "pjm-b654b4",
-      "locationSet": {"include": ["us"]},
+      "displayName": "PGE Energetyka Kolejowa",
+      "id": "pgeenergetykakolejowa-116894",
+      "locationSet": {"include": ["001"]},
```

```suggestion
      "locationSet": {"include": ["pl"]},
```
It looks like it only operates in Poland, not worldwide?

In addition, taginfo shows that there are still many elements are using "PKP Energetyka" as operator. It is recommended that you add a matchNames to ensure that the entry "PGE Energetyka Kolejowa" can be applied to all existing data.

---

Thanks. I limited to Poland and added `matchNames` in https://github.com/osmlab/name-suggestion-index/pull/9307/commits/4bef91e4e4ac28bcc43d78e2855b5770140d1f47